server/server.go

```diff
@@ -1464,6 +1464,7 @@ func (s *Server) ReplicateFileToMember(ctx context.Context, member *pdpb.Member,
 
 // PersistFile saves a file in DataDir.
 func (s *Server) PersistFile(name string, data []byte) error {
+	log.Info("persist file", zap.String("name", name), zap.Binary("data", data))
```

Comment:
Is it readable?

Reply:
Not readable. It will use base64. Because the API supports write arbitrary data, binary should be a good choice.

Reply:
why need to log the data？may be an md5 or checksum?

Reply:
It should not be very long. Sometime if we really want to debug, we can decode and check the content.